extensions/panache/hibernate-orm-panache/deployment/src/main/java/io/quarkus/hibernate/orm/panache/deployment/PanacheHibernateResourceProcessor.java

```diff
@@ -58,11 +56,6 @@ public final class PanacheHibernateResourceProcessor {
 
     private static final DotName DOTNAME_ID = DotName.createSimple(Id.class.getName());
 
-    @BuildStep
```

Comment:
What's the reason for removing this?

Reply:
It's a mistake. Thanks for the heads-up!

Reply:
👌🏼

Reply:
Fixed